packages/react-interactions/events/src/dom/create-event-handle/__tests__/useFocusWithin-test.internal.js

```diff
@@ -528,14 +541,17 @@ describe.each(table)(`useFocus`, hasPointerEvents => {
 
       suspend = true;
       await act(() => {
-        root.render(<Component />);
+        root2.render(<Component />);
       });
       expect(container2.innerHTML).toBe(
         '<div><input style="display: none;">Loading...</div>',
       );
       expect(onBeforeBlurWithin).toHaveBeenCalledTimes(1);
       expect(onAfterBlurWithin).toHaveBeenCalledTimes(1);
-      resolve();
+      await act(() => {
+        suspend = false;
+        resolve();
+      });
```

Comment:
Makes sense, the infinite loop error is because of the `root.render(null);` in the after each. If we flush here, it would be good to assert on the output:

```suggestion
      });
      expect(container2.innerHTML).toBe('<div><input style=""></div>');
```